pkg/virt-controller/services/template.go

```diff
@@ -1311,6 +1311,10 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 					Name:      "hook-sidecar-sockets",
 					MountPath: hooks.HookSocketsSharedDirectory,
 				},
+				{
+					Name:      "libvirt-runtime",
+					MountPath: "/var/run/libvirt",
+				},
```

Comment:
@andreabolognani I'm not 100% sure but I think so. If you are able to launch the VM you are able to exec into the compute container. Sidecar are disable and needs to be enabled through the feature gates so we can control this

Reply:
We could also add an additional annotation to make the libvirt-runtime volume optionally shareable if really necessary

Reply:
Avoiding the need to access libvirt will be the better option but currently not working. Not sure if some of the XML parsing is directly handled at libvirt level. If so, then we need access to libvirt sock

Reply:
My concern is that right now you have a trusted component (KubeVirt) in charge of talking to libvirt and hook sidecars can only perform relatively minor modifications to the VM configuration, but with your change you would have to trust the hook sidecar more because it would have basically free reign over anything that can be controlled via libvirt. Maybe that's fine, but it does seem like it would potentially open the door to some nasty consequences.

Reply:
Let's see what is the outcome of the virt-manger issue if we can avoid accessing libvirt this will be the best option